src/redmine/models/project.ts

```diff
@@ -3,4 +3,10 @@ export interface Project {
   name: string;
   description: string;
   identifier: string;
+  parent?: ProjectParentItem;
+}
+
+export interface ProjectParentItem {
+   id: string;
```

Comment:
Actually when I try to format my code with prettier, it formats whole file even tough I select the **format selection** option.

Reply:
Prettier is quite aggressive with the formatting since it "rebuilds" the file from scratch. So it first parses the whole file and then "stringifies" it back to the text, with formatting rules applied. It has one massive advantage - code formatting is the same for the file, regardless of the programmers feeling of style. Downside? It will always try to be smarter than you 😉